src/mono/wasm/runtime/loader/run.ts

```diff
@@ -409,11 +446,13 @@ async function createEmscriptenMain(): Promise<RuntimeAPI> {
     });
 
     init_globalization();
+
     // TODO call mono_download_assets(); here in parallel ?
     const es6Modules = await Promise.all(promises);
     initializeModules(es6Modules as any);
 
     await runtimeHelpers.dotnetReady.promise;
+    await invokeOnRuntimeReady(globalObjectsRoot.api);
```

Comment:
please don't block on this.

Reply:
Why shouldn't we wait on the promise? 

The use case is

"There is nuget package that wants to do something when runtime is ready, but before user code is run after dotnet.create"

Reply:
okay